tests/overrides/test_data_parallel.py

```diff
@@ -151,7 +151,6 @@ def __init__(self):
             super().__init__()
             self.layer = nn.Linear(2, 3)
 
-        @auto_move_data
```

Comment:
Unclear whether the whole test or just the decorator should have been removed. The language of the test sounds nonspecific to the `auto_move_data` decorator

Reply:
I'd say what you did is correct :)